internal/generate/kubernetes/generate.go

```diff
@@ -62,3 +103,141 @@ func execute(cmd *exec.Cmd) {
 		log.Printf("Failed to capture stderr: %v\n", errStderr)
 	}
 }
+
+func getAwsEnvars(awsSecrets Secrets) []string {
+	env := os.Environ()
+	env = append(env, fmt.Sprintf("AWS_ACCESS_KEY_ID=%s", awsSecrets.Aws.AwsAccessKeyID))
+	env = append(env, fmt.Sprintf("AWS_SECRET_ACCESS_KEY=%s", awsSecrets.Aws.AwsSecretAccessKey))
+	env = append(env, fmt.Sprintf("AWS_DEFAULT_REGION=%s", awsSecrets.Aws.Region))
+
+	return env
+}
+
+func readSecrets() Secrets {
+
+	dir := getCwd()
+
+	secretsFile := fmt.Sprintf("%s/secrets.yaml", dir)
+
+	data, err := ioutil.ReadFile(secretsFile)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	awsSecrets := Secrets{}
+
+	err = yaml.Unmarshal(data, &awsSecrets)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	return awsSecrets
+}
+
+func writeSecrets(s Secrets) {
+	secretsYaml, err := yaml.Marshal(&s)
+
+	fmt.Printf("SECRETS: %v", string(secretsYaml))
+
+	if err != nil {
+		log.Fatalf("error: %v", err)
+		panic(err)
+	}
+
+	dir, err := os.Getwd()
+
+	if err != nil {
+		log.Fatalf("Getting working directory failed: %v\n", err)
+		panic(err)
+	}
+
+	secretsFile := fmt.Sprintf("%s/secrets.yaml", dir)
```

Comment:
You can use `fikepath.Join` to create paths in a portable way

Reply:
Changed to `filepath.Join`